scripts/create-json-for-web.sh

```
@@ -0,0 +1,55 @@
+#
+# オプション:
+#   -w work_dir: 作業ディレクトリを指定する (デフォルト: 一時ディレクトリ)
+
+# check if poetry is installed
+if ! command -v poetry &> /dev/null; then
+    echo "poetry could not be found"
+    exit 1
+fi
+
+# cd to parent directory of this script
+cd $(dirname $(dirname $(realpath $0)))
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)